frontend/src/Pages/Student/Course/CourseResources.jsx

```
@@ -0,0 +1,97 @@
+                            )}
+
+                            <p className="text-sm text-gray-500 mt-2">
+                                <strong>Type:</strong> {resource.file_type} | <strong>Size:</strong>{" "}
+                                {(resource.file_size / 1024).toFixed(2)} KB
+                            </p>
+                        </div>
+
+                        <button
+                            onClick={() => handleDownload(resource.id, resource.file_name)}
+                            className="mt-4 sm:mt-0 sm:ml-4 inline-flex items-center bg-blue-600 hover:bg-blue-700 text-white px-4 py-2 rounded-lg text-sm font-medium shadow-md"
+                        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)